noether_simulator/package.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<package>
+  <name>noether_simulator</name>
+  <version>0.0.0</version>
+  <description>The noether_simulator package</description>
+  <maintainer email="alex.goins@swri.org">alex</maintainer>
```

Comment:
This should probably be format 2 package.xml to match everything else.

Maintainer should probably be Jorge

License is definitely Apache 2!

Reply:
I have pushed all the requested changes.